sample_dataset/eli5/load_eli5_dataset.py

```diff
@@ -0,0 +1,29 @@
+import os
+import pathlib
+
+from datasets import load_dataset
+
+
+def load_eli5_dataset():
+    # set file path
+    file_path = "MarkrAI/eli5_sample_autorag"
+
+    # load dataset
+    corpus_dataset = load_dataset(file_path, "corpus")['train'].to_pandas()
+    qa_dataset = load_dataset(file_path, "qa")['train'].to_pandas()
+
+    # path setting
+    sample_dataset_dir = pathlib.PurePath(__file__).parent.parent
+    project_dir = os.path.join(sample_dataset_dir, "eli5")
```

Comment:
You can just save this to `pathlib.Purepath(__file__)` directory.
Because `__file__` is load_eli5_dataset.py file path, so you can save qa.parquet and corpus.parquet at that directory,